scripts/install-qemu-user-static.sh

```diff
@@ -0,0 +1,8 @@
+#/bin/bash
+dpkg -l | grep qemu-user-static
```

Comment:
Please don't do that. I don't have `dpkg` or `qemu-aarch64` on my mac.

Reply:
This should be run only in case that the architecture aarch64 is detected.

Reply:
Is it possible to run mac on arm64?

Reply:
You are right and I am wrong with that argument that it will be executed only on case that aarch64 is detected. 
But I am not sure how this kind of thing is solved on mac...